rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -220,6 +222,7 @@ void emu_settings::SaveSettings()
 	m_config.seek(0);
```

Comment:
This will break opening/editing and saving config without starting a game

also m_config should be changed to not be a member variable if its going to be closed every time like this